docs/cache/manage/limits.md

```diff
@@ -27,61 +27,63 @@ The limits on this page are soft limits that can be altered unless specifically
-| ListFetch                | Number of elements in response, or 1 if list is not found       |
-| ListConcatenateBack      | Number of elements in request                                   |
-| ListConcatenateFront     | Number of elements in request                                   |
-| ListLength               | 1                                                               |
-| ListPopBack              | 1                                                               |
-| ListPopFront             | 1                                                               |
-| ListPushBack             | 1                                                               |
-| ListPushFront            | 1                                                               |
-| ListRemoveValue          | 1                                                               |
-| ListRetain               | 1                                                               |
-| SetAddElement            | 1                                                               |
-| SetAddElements           | Number of elements in request                                   |
-| SetFetch                 | Number of elements in response, or 1 if set is not found        |
-| SetRemoveElement         | 1                                                               |
-| SetRemoveElements        | Number of elements in request                                   |
-| SetContainsElement       | 1                                                               |
-| SetContainsElements      | Number of elements in request                                   |
-| SetLength                | 1                                                               |
-| SortedSetPutElement      | 1                                                               |
-| SortedSetPutElements     | Number of elements in request                                   |
-| SortedSetFetchByRank     | Number of elements in response, or 1 if sorted set is not found |
-| SortedSetFetchByScore    | Number of elements in response, or 1 if sorted set is not found |
-| SortedSetGetScore        | 1                                                               |
-| SortedSetGetScores       | Number of elements in request                                   |
-| SortedSetRemoveElement   | 1                                                               |
-| SortedSetRemoveElements  | Number of elements in request                                   |
-| SortedSetGetRank         | 1                                                               |
-| SortedSetIncrementScore  | 1                                                               |
-| SortedSetLength          | 1                                                               |
-| SortedSetLengthByScore   | 1                                                               |
+| API Name                 | Batch API | Operations                                                        |
```

Comment:
I also wonder if we're going to include the `/2` in this column if we want to call this `Weighted Operations` or something like that?

Reply:
i'm fine with Operations